app/assets/stylesheets/govuk_publishing_components/components/_big-number.scss

```
@@ -1,11 +1,12 @@
 .gem-c-big-number {
   margin-bottom: govuk-spacing(3);
-  font-family: $govuk-font-family;
+  @include govuk-typography-common($font-family: $govuk-font-family);
+  color: $govuk-text-colour;
```

Use the mixin – it'll take care of setting back to `#000000` when printing.

```suggestion
  @include govuk-text-colour;
```

---

This is what happens when I do a page find in the DS sass reference and then don't skip beyond the first item 🤦